tests/devices/unit_tests/test_undulator.py

```diff
@@ -95,6 +108,27 @@ async def test_length_not_propagated_if_not_supplied():
             "UND-01",
             name="undulator",
             poles=80,
+            id_gap_lookup_table_path=ID_GAP_LOOKUP_TABLE_PATH,
         )
     assert undulator.length is None
     assert "undulator-length" not in (await undulator.read_configuration())
+
+
+@pytest.mark.parametrize(
+    "dcm_energy, expected_output", [(5730, 5.4606), (7200, 6.045), (9000, 6.404)]
```

Comment:
Nit: This isn't `DCM` energy, it's just energy

Reply:
ok